contrib/init/sysvinit-redhat/docker

```diff
@@ -97,6 +99,13 @@ rh_status_q() {
     rh_status >/dev/null 2>&1
 }
 
+
+check_for_cleanup() {
+  if [ -f ${pidfile} ]; then
+    /bin/ps -fp $(cat ${pidfile}) > /dev/null || rm ${pidfile}
+  fi
```

Comment:
This seems a little hacky IMO, but at the very least this ought to have indentation that matches the rest of the file.